scripts/vscode-build.ps1

```diff
@@ -1,5 +1,6 @@
 param (
-  [Parameter(Mandatory = $true)][string]$filePath
+  [Parameter(Mandatory = $true)][string]$filePath,
+  [string]$msbuildEngine = "vs"
```

Comment:
This parameter effectively acts like a global variable here. The implementation of `InitializeBuildTool` below relies on reading this variable to pick the appropriate build engine. 

This pattern is how our core `build.ps1` script works and I'm essentially replicating it here. That should make this script more resilient over time as it should evolve in sync with our core build logic.

Reply:
And `InitializeBuildTool` is in arcade and hard coded to dotnet.exe :(